charts/cinder-csi-plugin/templates/snapshot-controller-rbac.yaml

```diff
@@ -0,0 +1,85 @@
+{{- if .Values.csi.snapshotController.enabled -}}
+# RBAC file for the snapshot controller.
+#
+# The snapshot controller implements the control loop for CSI snapshot functionality.
+# It should be installed as part of the base Kubernetes distribution in an appropriate
+# namespace for components implementing base system functionality. For installing with
+# Vanilla Kubernetes, kube-system makes sense for the namespace.
```

Comment:
May be its better to add comment here , enable this only if your Kubernetes distribution does not bundle the snapshot controller. It should be installed only once per cluster , independent of CSI driver.